### ISSUE TITLE
blaSHV carbapenemase activity?

### ISSUE THREAD

**Issue by cgonzalezz** [arpcard/amr_curation#105]
We have a concern about the drug classes listed for SHV-beta lactamases in CARD. We find "carbapenem" listed for the family as a whole, however the publications you list do not appear to indicate SHV having carbapenemase activity. We currently license CARD for use in software and provide drug metadata based on this drug information, and we have had software users raise this concern about SHV as well. Can you provide any insight into this drug classification or let us know if this is going to be updated in a future version of CARD?

**Comment by bpalcock** [comment]
Thank you for bringing this to our attention. This was a straightforward error in our curation and I've gone ahead and corrected it. We'll be releasing a new version of CARD in early August where this will no longer appear. 

I will add that sometimes the semantic relationship `confers_resistance_to_drug_class` is added to an AMR gene family (such as SHVs) when _some but not all_ members match that criteria. This is incorrect, or at least ambiguous, but arises because of pre-existing rules which we are currently planning to overhaul in the coming weeks for beta-lactamase curation. This is something we recently resolved for the OXA beta-lactamases in our February 2024 release, but hasn't yet been generalized for all Bla families. 

That was a bit tangential but I wanted to add some context as your concern is a known issue. Regardless, in this case it was simply an error for the blaSHV family and I have gone ahead and corrected it. I will leave this issue open until I have confirmed the fix has been made after the next release.

Thanks again,
Brian
